docs/tutorials/podman_pasta_ipv6_with_snid_ipv4.md

```diff
@@ -0,0 +1,643 @@
+
+`/etc/systemd/system/snid-server.service`:
+```
+[Unit]
+Description=SNID Server Service
+Requires=snid-routes.service
+
+[Service]
+User=snid
+Group=snid
+ExecStart=/bin/bash -c 'cd /opt/snid && ./snid -listen tcp:172.16.1.10:443 -mode nat46 -nat46-prefix 64:ff9b:1:: -backend-cidr 2001:db8:0000:0001:0000:0000:0001:0001/112'
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Reload Systemd Daemon
+```
+systemctl daemon-reload
+```
+
+Enable Services
+```
+systemctl enable --now snid-routes.service snid-server.service
+```
+
+Check for Errors:
+```
+systemctl restart snid-server.service
+systemctl status snid-server.service
+```
+
+
+## Multiple snid on Remote Servers
+In case where multiple `snid` Servers are deployed, it is possible to use several portions of the `64:ff9b:1::/48` Address Space (`0064:ff9b:0001:0000:0000:0000:0000:0000` ... `0064:ff9b:0001:ffff:ffff:ffff:ffff:ffff`), consistently with [RFC 8215](https://datatracker.ietf.org/doc/html/rfc8215#section-5).
+
+Example:
+- `snid1` Settings for `nat46-prefix`:
+  - SNID Server Address: `2001:db8:0000:0001:0000:0000:0001:0066`
+  - Minimized Network: `64:ff9b:1::1:0:0/96`
+  - Expanded Network: `0064:ff9b:0001:0000:0000:0001:0000:0000/96`
+  - First Address: `0064:ff9b:0001:0000:0000:0001:0000:0000`
+  - Last Address: `0064:ff9b:0001:0000:0000:0001:ffff:ffff`)
+- `snid2` Settings for `nat46-prefix`
+  - SNID Server Address: `2001:db8:0000:0001:0000:0000:0001:0067`
+  - Minimized Network: `64:ff9b:1::2:0:0/96`
+  - Expanded Network: `0064:ff9b:0001:0000:0000:0002:0000:0000/96`
+  - First Address: `0064:ff9b:0001:0000:0000:0002:0000:0000`
+  - Last Address: `0064:ff9b:0001:0000:0000:0002:ffff:ffff`)
+- `snid3` Settings for `nat46-prefix`: 
+  - SNID Server Address: `2001:db8:0000:0001:0000:0000:0001:0068`
+  - Minimized Network: `64:ff9b:1::3:0:0/96`
+  - Expanded Network: `0064:ff9b:0001:0000:0000:0003:0000:0000/96`
+  - First Address: `0064:ff9b:0001:0000:0000:0003:0000:0000`
+  - Last Address: `0064:ff9b:0001:0000:0000:0003:ffff:ffff`)
+- ...
+
+### Systemd Services
+Refer to the Previous section for Localhost or [this](https://github.com/luckylinux/podman-network-setup/).
+
+### Routes from Podman Host to snid Servers
+On the Podman Host a Route must be configured TO the `snid` Remote Servers in order to send back Traffic to the Client.
+```
+# Send back traffic to snid1
+ip -6 route replace 64:ff9b:1::1:0:0/96 via 2001:db8:0000:0001:0000:0000:0001:0066
+
+# Send back traffic to snid2
+ip -6 route replace 64:ff9b:1::2:0:0/96 via 2001:db8:0000:0001:0000:0000:0001:0067
+```
+
+### Routes from snid Servers to Podman Host
+On each `snid` Server a Route must be configured TO the `podman` Host in order for the original HTTP(s) Request to be forwarded correctly:
+```
+# Send traffic to Podman Host
+ip -6 route replace 2001:db8:0000:0001:0000:0000:ff15:0000/112 via 2001:db8:0000:0001:0000:0000:0000:0100
+```
```

Comment:
There is a better solution for this when the `snid`s share the same subnet as the services. You can use [`ndppd`](https://github.com/DanielAdolfsson/ndppd) to answer neighbor discovery requests.
Here is an example config:
```conf
proxy eth0 {
	router no
	rule 2001:db8:0000:0001:0000:0000:ff15:0000/112 {
		static
	}
}
```
In the case, where `snid` runs somewhere else, there is no need for this, since the routers will take care of it.